src/ape/utils.py

```diff
@@ -222,6 +243,39 @@ def available_plugins(self) -> Set[str]:
             if repo.name.startswith("ape-")
         }
 
+    def download_package(self, repo_path: str, version: str, target_path: Path):
```

Comment:
Kinda neat - this works if you don't have the Github auth token, since it's public repos

Reply:
Wait, we probably don't need the Auth token for detected 2nd class either?

Reply:
Wow you are right! We are totally erroring out early for no reason

Reply:
https://github.com/ApeWorX/ape/pull/287/files